notebooks/4_visualize_forecast.ipynb

```diff
@@ -0,0 +1,1082 @@
+    "    run_countries = list(set(list(org_dest.Origin) + list(org_dest.Destination)))\n",
+    "    for country in run_countries:\n",
+    "        country_count.loc[country] = country_count.loc[country] + 1\n",
+    "\n",
+    "    # identify first introductions to each country\n",
+    "    first_intro = org_dest.drop_duplicates(subset=[\"Destination\"])\n",
+    "    first_intros_all = first_intros_all.append(first_intro, ignore_index=True)\n",
+    "\n",
+    "    # identify first export from each country\n",
+    "    first_export = org_dest.drop_duplicates(subset=[\"Origin\"])\n",
+    "    first_exports_all = first_exports_all.append(first_export, ignore_index=True)\n",
+    "\n",
+    "    # COI first intros by origin\n",
+    "    coi_first_intros = (org_dest[org_dest[\"Destination\"] == coi]).drop_duplicates(\n",
+    "        subset=[\"Origin\"]\n",
+    "    )\n",
+    "    coi_first_intros_by_origin = coi_first_intros_by_origin.append(\n",
+    "        coi_first_intros, ignore_index=True\n",
+    "    )\n",
+    "\n",
+    "# Save all summaries\n",
+    "\n",
+    "org_dest_all.to_csv(f\"{out_dir}/summary_stats/{run_name}/org_dest_all.csv\", index=False)\n",
+    "country_count.reset_index().to_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/country_count.csv\", index=False\n",
+    ")\n",
+    "first_intros_all.to_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/first_intros_all.csv\", index=False\n",
+    ")\n",
+    "first_exports_all.to_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/first_exports_all.csv\", index=False\n",
+    ")\n",
+    "coi_first_intros_by_origin.to_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/coi_first_intros_by_origin.csv\", index=False\n",
+    ")\n",
+    "countries_firstintro.reset_index().to_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/countries_firstintro.csv\", index=False\n",
+    ")\n",
+    "countries_reintros.reset_index().to_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/countries_reintros.csv\", index=False\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you have already created the summary dataframes, you can restart here: \n",
+    "\n",
+    "org_dest_all = pd.read_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/org_dest_all.csv\"\n",
+    ")\n",
+    "country_count = pd.read_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/country_count.csv\"\n",
+    ")\n",
+    "first_intros_all = pd.read_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/first_intros_all.csv\"\n",
+    ")\n",
+    "first_exports_all = pd.read_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/first_exports_all.csv\"\n",
+    ")\n",
+    "coi_first_intros_by_origin = pd.read_csv(\n",
+    "    f\"{out_dir}/summary_stats/{run_name}/coi_first_intros_by_origin.csv\"\n",
+    ")\n",
+    "countries_firstintro = pd.read_csv(\n",
```

Comment:
Add index_col=0 during csv import for countries_firstintro and countries_reintros to fix issue with old index column.

countries_firstintro = pd.read_csv(
    f"{out_dir}/summary_stats/{run_name}/countries_firstintro.csv", index_col=0
)
countries_reintros = pd.read_csv(
    f"{out_dir}/summary_stats/{run_name}/countries_reintros.csv", index_col=0
)